examples/DateRangePickerWrapper.jsx

```diff
@@ -129,6 +132,7 @@ class DateRangePickerWrapper extends React.Component {
           focusedInput={focusedInput}
           startDate={startDate}
           endDate={endDate}
+          isRTL={props.isRTL}
```

Comment:
please destructure `isRTL` out of `this.props` at the top of the function; however, in this case, it will just be passed through via `props` so there's no need to destructure it at all.